src/shared/internal-calendar/internal-calendar.style.tsx

```diff
@@ -100,20 +95,22 @@ export const OptionsOverlay = styled.div<OverlayStyleProps>`
     }}
 `;
 
-export const Header = styled.div`
+export const Header = styled.div<GeneralStyleProps>`
     display: flex;
     justify-content: space-between;
-    margin-bottom: 0.5rem;
-`;
+    margin: 0 0.5rem 1rem 0.5rem;
```

Comment:
i think Container can deal with the horizontal spacing which is 1.25rem for input and 2rem for standalone, header just needs to deal with the bottom margin

Reply:
actually some spacing and styles have deviated from the current Figma. better to address in another ticket.

Reply:
will deal with it in the refactor to have CalendarWrapper